### DISCUSSION TITLE
Translations in content lists/assignments – picking a main/base language

### DISCUSSION THREAD

**Discussion by cvogl** [directus/directus#15661] [Archive]
Hi all!

Trying to setup a multilingual site with Directus and a NextJS-Frontend, I'm quite happy with – even impressed by – Directus' general abilities.

I'm only running into two issues with the translations in content lists / content assignment views. Maybe someone ran into the same & succeeded in resolving them or can point me in a right direction?

_Problem:_
Content items are listed with *all* available languages

_Question:_
Is it possible to pick just one language for content lists & assignment views?

For the sake of simplicity of explanation / reproducibility, I'll demonstrate it with for your official »Directus Next.js Example«, https://github.com/directus/examples/tree/main/nextjs

I've added a translation field to the articles collection and some fields in that, among these, of course, title and body:

![Model-Articles-Translations](https://user-images.githubusercontent.com/971093/191240098-3026c791-e07c-4856-9112-5674298e63db.png)

Unfortunately, in lists there does not seem to be a possibility to choose the title in only one language – there are always listed all translations for a field (the right „Title“ field in the screenshot; the left Title is the original field in the examples collection Articles)

![Articles-List](https://user-images.githubusercontent.com/971093/191240510-474300bb-bf00-4af3-b9e0-bae031e2a6b1.png)

Obviously, I could place one language in the „meta collection“ (which is already the case in the Example, that's why I left it in the screenshot) – but this complicates data fetching and/or frontend components by (demanding an extra layer for collecting content data). Or I could force users to duplicate the needed information for the views (by entering a base translation on content creation as well as in a translation). 

But I'd like to avoid that in favor of a clean data model – it seems more natural to address this backend issue in the backend views.

Any suggestions? Should I look into Custom Interfaces? Or maybe Hooks?

**Comment by azrikahar** [comment]
You can use the `translations` field with Translations display to achieve this 👍 Selecting `translations -> title` instead will indeed show all as an array as it's based on the `title` field's display (by default not configured, so it'll show all), not the `translations` field's display (which can use the unique Translations display).

Here's a quick example on what it would look like (also showcased `translations -> title` just to mirror your setup briefly):

https://user-images.githubusercontent.com/42867097/193990428-7daa94b3-962f-4512-8fb3-1b0950ac402c.mp4

**Reply by cvogl** [reply]
Great – thank you very much, Azri!

I swear I've been there already while trying to figure it out, but in my more complicated setup involving M2A relations.
I still fail with the same assignment, but I'll further look into that issue myself. I'll share my solution to that here as soon as I have it.

Thanks again,
and have a good week!

**Comment by Rar9** [comment]
Is it possible to also change the TRANSLATIONS to be called TITLE ? 
![image](https://user-images.githubusercontent.com/2239608/194013950-bc35d5c9-4d46-4038-a605-21af0ebcc947.png)

Your Video Clips are just PERFECT to get started and you should include this to Directus DOCs !!!

**Reply by cvogl** [reply]
Yes, Rar9,
you can do that via ›Field translations‹ for the translation relation. 
I've recorded a short demonstration:

[Title_Label.webm](https://user-images.githubusercontent.com/971093/194047413-dce35b4c-8fe3-499b-8f3b-5c7f045af5ff.webm)

Regards,
Christian

**Comment by Rar9** [comment]
Thanks but then these translatiosn cant be searched by as its grayed out
![image](https://user-images.githubusercontent.com/2239608/194052513-185115d6-1e32-4f80-a492-0f086edab135.png)

**Reply by cvogl** [reply]
Yes, I also encountered that.
Field label translations are also available for the fields in your translation collection (in my example: ›articles_translations‹); but with those fields, we run into other issues as you & @azrikahar already pointed out in your issue #15849 .

I have not yet found a solution (or workaround).
